<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>auth</class>
 <widget class="QWidget" name="auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wishcord - auth</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QPushButton" name="pushAction">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePass">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>jean</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineLogin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>jean</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePseudo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPseudo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Pseudo</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPass">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSwitch">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelState">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushSwitch</sender>
   <signal>clicked()</signal>
   <receiver>auth</receiver>
   <slot>switchAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushAction</sender>
   <signal>clicked()</signal>
   <receiver>auth</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>switchAction()</slot>
  <slot>action()</slot>
 </slots>
</ui>
